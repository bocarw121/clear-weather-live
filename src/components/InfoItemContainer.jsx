import React from "react";
import { StyleSheet, View } from "react-native";

const InfoItemContainer = ({ children }) => {
  return <View style={styles.container}>{children}</View>;
};

export default InfoItemContainer;

const styles = StyleSheet.create({
  container: {
    flexDirection: "row",
    justifyContent: "space-evenly",
    flexWrap: "wrap",
    marginTop: 50,
    marginBottom: 50,
  },
});
