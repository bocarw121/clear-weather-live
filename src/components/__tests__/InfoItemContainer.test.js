import React from "react";

import { render } from "@testing-library/react-native";

import InfoItemContainer from "../InfoItemContainer";

it("Should render the InfoItemContainer", () => {
  const screen = render(<InfoItemContainer />);

  expect(screen).toMatchSnapshot();
});
