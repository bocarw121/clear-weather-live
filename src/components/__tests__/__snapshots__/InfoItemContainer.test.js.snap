// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the InfoItemContainer 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "flexWrap": "wrap",
      "justifyContent": "space-evenly",
      "marginBottom": 50,
      "marginTop": 50,
    }
  }
/>
`;
